lib/ansible/modules/windows/win_firewall_profile.ps1

```diff
@@ -0,0 +1,118 @@
+#!powershell
+
+# Copyright : (c) 2019, Simon Lecoq <simon.lecoq@live.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+#Requires -Module Ansible.ModuleUtils.CamelConversion
+
+# Specs and init
+$spec = @{
+    options = @{
+        profiles = @{ type = "list"; elements = "str"; choices = "Domain", "Private", "Public"; default = @( "Domain", "Private", "Public" ); aliases = @( "profile" ) }
+        enabled = @{ type = "str"; choices = "True", "False", "NotConfigured" }
+        default_inbound_action = @{ type = "str"; choices = "NotConfigured", "Allow", "Block"; aliases = @( "inbound" ) }
+        default_outbound_action = @{ type = "str"; choices = "NotConfigured", "Allow", "Block"; aliases = @( "outbound" ) }
+        log_file_name = @{ type = "path"; aliases = @( "log_file" ) }
```

Comment:
`log_file_name` doesn't really match up here as it looks like it is the path of the log file. I would just have it as `log_file` without the alias.